### DISCUSSION TITLE
[Feature]: Delete photos from albums

### DISCUSSION THREAD

**Discussion by oddlama** [immich-app/immich#1655] [Feature Request]
### Feature detail

I currently see no way to delete a photo from an album in the web app, except by deleting the whole photo. When I accidentally add a photo, I'd like to be able to remove it afterwards. But it may very well be that I'm simply not finding the option.

### Platform

Web

**Comment by alextran1502** [comment]
You can perform this action by choosing the select icon when your mouse hovers on the photo in an album

![image](https://user-images.githubusercontent.com/27055614/192116472-89f97b15-d2c0-4ea9-a7fb-f1564d8020fa.png)

**Comment by oddlama** [comment]
Oh I'm sorry I wasn't specific enough. The delete button is missing when I add a photo to an album that was shared with me by another person. It makes sense that I cannot delete the images that others have added, but I also cannot remove images from the album which I added and which are technically owned by me.

**Comment by bo0tzz** [comment]
#265 will cover implementing shared albums more broadly, including a permissions system for adding/deleting assets. I think that might cover this case as well?

**Comment by CloneXpert** [comment]
I'm often missing this feature too.
My desired workflow would be the following:
I create an album where I add all the photos and slide through the assets and with a shortcut I could just remove the current asset from that album.
Note that currently it's only possible to remove it from the grid view with the photos selected. I couldn't see the menu item available with the single asset view mode.
